assets/src/edit-story/components/panels/textStyle.js

```diff
@@ -108,22 +110,30 @@ function TextStylePanel({ selectedElements, onSetProperties }) {
     lineHeight,
     padding,
     backgroundColor,
+    color,
   });
   const [lockRatio, setLockRatio] = useState(true);
   useEffect(() => {
-    setState({ textAlign, letterSpacing, lineHeight, padding });
+    setState((oldState) => ({
+      ...oldState,
```

Comment:
This confused me quite a while. But these `setState`'s cleared all non-listed properties and that screwed things up completely. But I finally found this bug. Similarly a few lines below.